kvmagent/kvmagent/plugins/storage_device.py

```
@@ -1242,7 +1242,8 @@ def get_disk_by_wwid(wwid):
             if wwid not in multipath_topology:
                 disk = get_disk_by_wwid(wwid)
                 if disk is not None:
-                    rsp.devices.update({wwid: [Device(disk, "active")]})
+                    rsp.devices.update({wwid: [Device(disk, "active", convert_state(linux.read_file("/sys/block/%s/device/state" %
```

Pylint (3.3.7)

[convention] Line too long (130/100)

(C0301)